torch/_refs/__init__.py

```diff
@@ -680,6 +682,10 @@ def tan(a):
 def tanh(a):
     return prims.tanh(a)
 
+@_make_elementwise_unary_reference(ELEMENTWISE_TYPE_PROMOTION_KIND.INT_TO_FLOAT,
+                                   aten_op=None)
+def bfloat16(a):
```

Comment:
`bfloat16(a, *, memory_format=torch.preserve_format)`?

Reply:
It's OK to `raise NotImplementedError` if `memory_format` is not equal to `torch.preserve_format` for now